torch/__init__.py

```diff
@@ -1586,6 +1589,37 @@ def __call__(self, model_, inputs_):
         return self.compiler_fn(model_, inputs_, **self.kwargs)
 
 
+################################################################################
+# Exposing torch.export() and related support APIs
+################################################################################
+from ._constraint import Constraint
```

Comment:
This is not ok in terms of public API. Public APIs should come from public files or you should update their `__module__`.